pkg/upgrade/container/container.go

```diff
@@ -110,6 +110,11 @@ func New(name string, spec upgradeapiv1.ContainerSpec, opt ...Option) corev1.Con
 		}}, spec.Env...),
 		EnvFrom: spec.EnvFrom,
 	}
+
+	for _, v := range spec.VolumeMounts {
+		container.VolumeMounts = append(container.VolumeMounts, v)
+	}
+
```

Comment:
This isn't enough because you can reference volumes by name all day but if they do not exist for the pod ...